src/aws-util/Cargo.toml

```diff
@@ -20,7 +20,7 @@ bytes = "1.3.0"
 bytesize = "1.1.0"
 http = "1.1.0"
 hyper-tls = "0.5.0"
-mz-ore = { path = "../ore", default-features = false }
+mz-ore = { path = "../ore", default-features = true }
```

Comment:
can we list the specific features required instead? just enabling default-features is going to break cloud, because it'll make aws-util start pulling in the workspace-hack crate

Reply:
Ah yeah this was just part of the revert commit. I'll just back this out. Doesn't seem to be required.